readme.txt

```diff
@@ -88,11 +88,13 @@ If you are still seeing this message after a few days, feel free to get in touch
 
 = Why aren’t any ads appearing on my site after I connected AdSense? =
 
-If you’re new to AdSense when you connect via Site Kit, your new AdSense account and your site will need to be manually reviewed and approved for ads by the AdSense team. Ads will not display until your account and site have been approved. [Check out this guide for more information about the approval process and timeline.](https://support.google.com/adsense/answer/76228)
+If you're new to AdSense when you connect via Site Kit, your new AdSense account and your site will need to be manually reviewed and approved for ads by the AdSense team. Ads will not display until your account and site have been approved. [Check out this guide for more information about the approval process and timeline.](https://support.google.com/adsense/answer/76228)
 
-You can check your approval status in Site Kit by going to Settings > Connected Services > AdSense and clicking “Check your site status.” This link will direct you to AdSense. If you see “Ready,” your account and site have been approved and should be displaying ads. If you see “Getting ready…,” your account and site are still under review and your site will not display ads until they have been approved.
+You can check your approval status in Site Kit by going to **Settings > Connected Services > AdSense** and clicking **Check your site status**. This link will direct you to AdSense. If you see "Ready," your account and site have been approved and should be displaying ads. If you see "Getting ready...," your account and site are still under review and your site will not display ads until they have been approved.
```

Comment:
Looks like some stylized characters were lost in the update here, can we please preserve those?

Reply:
Thanks - I didn't request permission to the docs since the IB and the discussion had the ready copy! I've re-added the "" (quotes) around Check your site status. Not sure if that is in the copy or not - it was in the previous copy.